src/components/button/button.jsx

```diff
@@ -24,13 +29,9 @@ module.exports = React.createClass({
   },
 
   renderAnchor: function() {
-    var props = {
-      id: this.props.id,
-      className: classNames('component-button', this.props.size, this.props.purpose),
-      href: this.props.href,
-      target: this.props.target,
-      onClick: this.props.handleClick
-    };
+    const props = _.omit(this.props, 'children');
```

Comment:
Caught up with "The Great Cameron" on this one and the reference is still the same so it makes sense to keep this as a const